apps/web/src/app/file-explorer/file-explorer-table.tsx

```
@@ -0,0 +1,278 @@
+    const folders = data.filter((item) => item.type === "folder");
+    const files = data.filter((item) => item.type === "file");
+
+    return (
+        <div className="w-full">
+            {/* Toggle Button */}
+            <div className="flex justify-end mb-4">
+                <button
+                    onClick={toggleViewMode}
+                    className="p-2 rounded-md hover:bg-accent transition-colors"
+                    aria-label={`Switch to ${viewMode === "table" ? "grid" : "table"} view`}
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)